docs/core/compatibility/aspnet-core/5.0/kestrel-libuv-transport-obsolete.md

```diff
@@ -8,9 +8,9 @@ ms.date: 10/01/2020
 
 Earlier versions of ASP.NET Core used Libuv as an implementation detail of how asynchronous input and output was performed. In ASP.NET Core 2.0, an alternative, <xref:System.Net.Sockets.Socket>-based transport was developed. In ASP.NET Core 2.1, Kestrel switched to using the `Socket`-based transport by default. Libuv support was maintained for compatibility reasons.
 
-At this point, use of the `Socket`-based transport is far more common than the Libuv transport. Consequently, Libuv support is marked as obsolete in .NET 5.0 and will be removed entirely in .NET 6.0.
+At this point, use of the `Socket`-based transport is far more common than the Libuv transport. Consequently, Libuv support is marked as obsolete in .NET 5 and will be removed entirely in .NET 6.0.
```

Comment:
Should we also change 6.0 to 6?

Reply:
I should have mentioned - .NET 6 is covered in #26842.